source/agora/common/Config.d

```diff
@@ -244,6 +244,12 @@ public struct ValidatorConfig
     /// The seed to use for the keypair of this node
     public immutable KeyPair key_pair;
 
+    /// The seed of PreImageCycle
+    public Hash cycle_seed;
+
+    /// The height of the seed of PreImageCycle
+    public Height cycle_seed_height;
+
```

Comment:
Is this used ? I don't see it being parsed and validated.

Reply:
It's used here and look into it more about validation.
https://github.com/linked0/agora/blob/06be326146d4da608fc6ea229088e592bfac9be4/source/agora/test/Base.d#L2087-L2101
 If you meant adding the items into the configuration file, It could be in another PR.

Reply:
I see, I was expecting that it would be parsed from the configuration file. I guess that makes sense to delay it, just add a comment saying this is currently not parsed ?

Reply:
OK, I will.

Reply:
Updated.